core/scss/components/card/settings.scss

```diff
@@ -1,6 +1,9 @@
+// THESE DON'T WORK
```

Comment:
What's the meaning of this comment?

Reply:
we need to talk about this .. if I make changes here nothing happens.  Glad you caught the comment because I forgot about it